myuw/templates/handlebars/card/alumni.html

```diff
@@ -0,0 +1,16 @@
+{% load templatetag_handlebars %}
+    {% tplhandlebars "alumni_card_content" %}
+    <div class="myuw-card">
+        <h3 class="myuw-card-title" aria-hidden="true">Alumni and Alumni Association</h3>
```

Comment:
@DannyNineNineNine these shouldn't use the `aria-hidden="true"` attribute. That will hide this header from screenreaders.